pkg/logs/read.go

```diff
@@ -0,0 +1,103 @@
+package logs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/observatorium/up/pkg/auth"
+	"github.com/observatorium/up/pkg/options"
+	"github.com/observatorium/up/pkg/transport"
+	"github.com/observatorium/up/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/prompb"
+)
+
+func Read(
```

Comment:
Let's add documentation comments for the exported types

Reply:
I would save this for later when we have proper interface for the read/write/query functions across metrics and logs.

Reply:
I don't think we need to wait for the interface; imo it's always good practice to document exported types. Let's just add a comment describing the current behavior